package.json

```diff
@@ -43,6 +43,7 @@
     "react-dom": "18.2.0",
     "react-hook-form": "^7.32.2",
     "react-icons": "^4.4.0",
+    "react-infinite-scroll-component": "^6.1.0",
```

Comment:
@DavidOB1 In retrospect we were so dumb with this, we should rework this too in a future ticket

Reply:
no u